### PR TITLE
Created new file for 0.7.0 news, framework in place

### PR DESCRIPTION
**Pull request by beccabau**
Signed-off-by: beccabau <rbau@us.ibm.com>

This PR is for https://github.com/eclipse/codewind/issues/1240

### CODE REVIEW

**Comment by beccabau** [comment]
@sishida can you merge changes so we can see them? thanks!

File: docs/_news/news07.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+duration: 1 minute
+permalink: news07
+---
+
+## Codewind 0.7.0
+// DOUBLE CHECK -- Monday 16 December 2019
+
+#### :sparkles: Remote Deploy with Codewind: Tech Preview :sparkles:
+
+You can now develop your code locally and securetly build and deploy your apps in the cloud! Why is this cool?
```
Comment (sishida) [inline docs/_news/news07.md]: Typo: "securetly"

File: docs/_news/news07.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- Free up local resources. :clap:
+- It's secure! We use [Keycloak](https://keycloak.org/) to secure the connection between your local editor and remote cloud deployment.
+
+// Curious? [Try it out!](link to docs) or [learn more](Link to our tech preview blog)
+
+#### New Features and Highlights for 0.7.0
```
Comment (sishida) [inline docs/_news/news07.md]: In the docs, we've been using lower case for everything except for the first word of the title and names of things. I'm OK with leaving the capitalization like this for the news--as long as we keep the news pages titles consistent with themselves. :)

File: docs/_news/news07.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+**OpenShift Do (odo)**
+- You can now easily import existing projects into Codewind and [use odo](https://github.com/eclipse/codewind/issues/1115) to build and deploy projects to your OpenShift or OKD cluster
+- Introducing [support for Java template](https://github.com/eclipse/codewind/issues/450) and Java projects can build and deploy [once configured](//permalink to landing page)
+- You no longer have to create your [docker registry secret](https://github.com/eclipse/codewind/issues/665) in Codewind
```
Comment (sishida) [inline docs/_news/news07.md]: We could probably use periods at the end of each of these. The List of Fixes are fragments, though, and can go without periods. :)

File: docs/_news/news07.md (hunk 2 above)
Comment (beccabau) [reply docs/_news/news07.md]: Thanks for the advice!

File: docs/_news/news07.md (hunk 1 above)
Comment (beccabau) [reply docs/_news/news07.md]: addressed in new commit.

File: docs/_news/news07.md (hunk 3 above)
Comment (beccabau) [reply docs/_news/news07.md]: addressed in new commit

**Pull request merged**